operations.py

```diff
@@ -0,0 +1,60 @@
+import logging
+import json
+import time
+
+try:
+    import dnstwist
+    import queue
+
+    MODULE_DNSTWIST = True
+except ImportError:
+    MODULE_DNSTWIST = False
+
+from connectors.core.connector import Connector, get_logger, ConnectorError
+
+logger = get_logger('dnstwist')
+
+THREAD_COUNT = 4
+
+
+def search(config, params):
+    try:
+        # fuzz = dnstwist.Fuzzer(params.get("domain").strip())
+        # TODO use at version 20211204
+        fuzz = dnstwist.DomainFuzz(params.get("domain").strip())
+        fuzz.generate()
+
+        jobs = queue.Queue()
+        for j in fuzz.domains:
+            jobs.put(j)
+
+        threads = []
+        for _ in range(THREAD_COUNT):
+            # worker = dnstwist.Scanner(jobs)
```

Comment:
Lets try using ThreadPoolExecutor class where you can provide a thread pool. 
with future.complete option you would not need run and join 

https://docs.python.org/3/library/concurrent.futures.html#threadpoolexecutor-example